src/main/java/org/webonise/polygon/Application.java

```diff
@@ -0,0 +1,118 @@
+package org.webonise.polygon;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Application {
+    private final PolygonCalculator polygonCalculator;
+    private final Scanner input;
+    private static final int MIN_SIDES = 3;
+    private static final int MAX_SIDES = 10;
+    private final Logger logger;
+
+    public Application() {
+        logger = Logger.getLogger(Application.class.getName());
+        polygonCalculator = new PolygonCalculator();
+        input = new java.util.Scanner(System.in);
+    }
+
+    public void start() {
+
+        try {
+            startApplication();
+        } catch (IllegalArgumentException illegalArgumentException) {
+            logger.log(Level.INFO, illegalArgumentException.getMessage());
+        } catch (InputMismatchException inputMismatchException) {
+            logger.log(Level.INFO, inputMismatchException.getMessage());
+        }
```

Comment:
As you are doing nothing special for both the exceptions you can catch parent Exception. Or else you can use multi catch using pipe character .